tests/dummy/config/environment.js

```diff
@@ -18,7 +18,8 @@ module.exports = function(environment) {
       // when it is created
     },
 
-    unstableApiURL: 'http://simonihmig.github.io/ember-bootstrap/'
+    unstableApiURL: 'http://simonihmig.github.io/ember-bootstrap/',
+    bootstrapVersion: process.env.BOOTSTRAPVERSION || 3
```

Comment:
This is the location where a user would define the version they want to use. Leave that simply as a number and put the environment variable override in the `if(environment ===` blocks. If users aren't supposed to use this, keep it away from the areas we expect them to modify.